readme.md

```diff
@@ -54,14 +54,14 @@ Returns a result object.
 
 ##### cwd
 
-Type: `string`  
+Type: `string`
 Default: `.`
 
 Directory to start looking for a package.json file.
 
 ##### normalize
 
-Type: `boolean`  
```

Comment:
Hah, yeah. You could always do it as a plugin ;)

https://github.com/sindresorhus/atom-editorconfig